lightning/src/routing/router.rs

```diff
@@ -178,17 +178,17 @@ impl< G: Deref<Target = NetworkGraph<L>> + Clone, L: Deref, ES: Deref, S: Deref,
 	fn create_blinded_paths<
 		T: secp256k1::Signing + secp256k1::Verification
 	> (
-		&self, recipient: PublicKey, peers: Vec<PublicKey>, secp_ctx: &Secp256k1<T>,
+		&self, recipient: PublicKey, recipient_tlvs: Option<RecipientData>, peers: Vec<PublicKey>, secp_ctx: &Secp256k1<T>,
```

Comment:
Hmm... not sure if we need these in payment paths. @valentinewallace Don't we use some other mechanism for including custom data there?